tests/phpunit/admin/class-wc-rest-stripe-settings-controller-test.php

```
@@ -563,6 +563,34 @@ public function provide_test_dismiss_notice() {
+		delete_option( 'wc_stripe_show_ocs_ap_banner' );
+
+		$request = new WP_REST_Request( 'POST', '/wc/v3/wc_stripe/settings/notice' );
+		$request->set_param( 'wc_stripe_show_ocs_ap_banner', true );
+
+		$response = $this->controller->dismiss_notice( $request );
+
+		$this->assertSame( 200, $response->get_status() );
+		$this->assertSame( 'no', get_option( 'wc_stripe_show_ocs_ap_banner' ) );
+
+		delete_option( 'wc_stripe_show_ocs_ap_banner' );
+	}
```

PHPMD (2.15.0)

[error] The method test_dismiss_notice_sets_ocs_ap_banner_to_no is not named in camelCase. (undefined)

(CamelCaseMethodName)